src/components/specs/mage/fire/mythic-plus/gameplay/damaging/FirePlaystyle.tsx

```diff
@@ -0,0 +1,66 @@
+import Section from '@shared/Section'
+import ItemLink from '@shared/Links/ItemLink'
+import Paragraph from '@shared/Paragraph'
+import spells from '@shared/spells.json'
+import items from '@shared/items.json'
+import SpellRotationElement from '@shared/spell-rotation/SpellRotationElement'
+import SpellRotationIcon from '@shared/spell-rotation/SpellRotationIcon'
+import SpellRotation from '@shared/spell-rotation/SpellRotation'
+// import { HiOutlineChartSquareBar } from 'react-icons/hi'
+
+function FaelineStomp() {
```

Comment:
Rename the function, as it's probably not about Faeline x)

Reply:
✅